src/events/IonEventStream.ts

```diff
@@ -1,59 +1,73 @@
 /*!
  * Copyright 2012 Amazon.com, Inc. or its affiliates. All Rights Reserved.
- *  
+ *
  * Licensed under the Apache License, Version 2.0 (the "License").
  * You may not use this file except in compliance with the License.
  * A copy of the License is located at
- *  
+ *
  *     http://www.apache.org/licenses/LICENSE-2.0
- *  
+ *
  * or in the "license" file accompanying this file. This file is distributed
  * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
 
-import {Reader} from "./IonReader";
+import {Reader} from "../IonReader";
 import {IonEvent, IonEventFactory, IonEventType} from "./IonEvent";
-import {Writer} from "./IonWriter";
-import {IonType} from "./IonType";
-import {IonTypes} from "./IonTypes";
-import {makeReader} from "./Ion";
+import {Writer} from "../IonWriter";
+import {IonType} from "../IonType";
+import {IonTypes} from "../IonTypes";
+import {defaultLocalSymbolTable, makeReader, ReaderOctetBuffer} from "../Ion";
+import {BinaryReader} from "../IonBinaryReader";
+import {BinarySpan} from "../IonSpan";
+import {ComparisonResultType} from "../ComparisonResult";
+import {ComparisonResult} from "../ComparisonResult";
 
 export class IonEventStream {
-    private eventStream: IonEvent[];
+    eventStream: IonEvent[];
```

Comment:
1. I had to make `eventStream` and `isEventStream` public to use it in test-driver code so that I could separate out the test-driver code from `IonEventStream`.

2. I think that makes sense will change it to events.

Reply:
> I had to make eventStream and isEventStream public to use it in test-driver code so that I could separate out the test-driver code from IonEventStream.

I think you can use `eventStream` already by calling the public `getEvents()` method to access it. Similarly, you could add a public `isEventStream()` method to access `isEventStream`.

Reply:
But in the case of embedded stream I want to modify the `eventStream` elements I can use a method called `setEvents()` and make `eventStream` private again and similarly create `isEventStream()`.  Does this sound good?